charts/tfy-distributor/values.yaml

```
@@ -0,0 +1,168 @@
+## Global configs
+global:
+  ## @param global.gateway the gateway URL for the application
+  gateway: <gateway-placeholder>
+
+## Distributor configs
+distributor:
+  resources:
+    limits:
+      ## @param distributor.resources.limits.cpu The maximum amount of CPU the service can use
+      cpu: 0.1
+      ## @param distributor.resources.limits.memory The maximum amount of memory the service can use
+      memory: 150M
+    requests:
+      ## @param distributor.resources.requests.cpu The amount of CPU the service requests
+      cpu: 0.1
+      ## @param distributor.resources.requests.memory The amount of memory the service requests
+      memory: 150M
+
+  ## @param distributor.serviceStreamReplicas Number Number of replicas for the service stream
+  serviceStreamReplicas: 2
```

```suggestion
  serviceStreamReplicas: 3
```

It can be either 3 or 5, we do not recommend 1. It is also related to the number of nodes in the NATS cluster.